content/hi/contribute/_index.md

```diff
@@ -38,7 +38,7 @@ menu:
 
 ![नया इशू](/images/how-to/howto-03.png)
 
-कृपया CNCF Slack पर #glossary चैनल से जुड़ें और @Catherine Paganini, @jmo, और @Seokho Son को सुचना दें कि आपने एक इशू जमा कर दी है और उस पर काम करना चाहते हैं। वे आपको वह इशू सौंपेंगे जो अन्य सभी को संकेत देगा कि यह शब्द पहले ही लिया जा चुका है।
+कृपया CNCF Slack पर [#glossary](https://cloud-native.slack.com/archives/C02TX20MQBB) चैनल से जुड़ें और @Catherine Paganini, @jmo, और @Seokho Son को सुचना दें कि आपने एक इशू जमा कर दी है और उस पर काम करना चाहते हैं। वे आपको वह इशू सौंपेंगे जो अन्य सभी को संकेत देगा कि यह शब्द पहले ही लिया जा चुका है।
```

Comment:
You should also replace the reviewers/approver here. It should be @Garima-Negi @sayantani11 @jayesh-srivastava @anubha-v-ardhan

Reply:
Should we link to `#glossary-localization-hindi` since this is the Hindi l10n for glossary?

Reply:
> Should we link to `#glossary-localization-hindi` since this is the Hindi l10n for glossary?

I agree. @Garima-Negi ??